homeassistant/components/sma/__init__.py

```diff
@@ -1 +1,40 @@
-"""The sma component."""
+"""The sma integration."""
+import asyncio
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, PLATFORMS
+
+
+async def async_setup(hass: HomeAssistant, config: dict) -> bool:
+    """Set up the sma component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up sma from a config entry."""
+    hass.data[DOMAIN][entry.entry_id] = entry.data
```

Comment:
You should't need to store this as its already being passed by `async_forward_entry_setup`

Reply:
Thanks, removed.